requirements.txt

```diff
@@ -19,3 +19,4 @@ six
 trimesh>=3.5.20
 sphinx==1.8.2
 sphinx_rtd_theme
+repoze.lru
```

Comment:
It needs only for python2.7. So rewrite it to `repoze.lru;python_version<"3.2"`.

Reply:
fixed in https://github.com/iory/scikit-robot/pull/181/commits/25cc3d02aa0309727d461758d0bb62043a73a95e